src/containers/other/NFTView/elements/DomainSettings/elements/DomainSettingsBody/DomainSettingsBody.tsx

```diff
@@ -91,7 +91,7 @@ export const DomainSettingsBody: React.FC<DomainSettingsBodyProps> = ({
 					<TextInputWithTopPlaceHolder
 						topPlaceholder={'Subdomain Name'}
 						onChange={localActions.setDomain}
-						text={localState.domain}
+						text={domain?.name}
```

Comment:
I don't think it is required here. If we wanted to display ```boat.cats.dogs.trees``` rather than ```wilder.boat.cats.dogs.trees```, then parsing it would be necessary. Domain name is fetched from the following in useZnsDomain.tsx: 

```
const domain = formatDomain(await sdk.getDomainById(id));
```

Screenshot shows the subdomain name without any modification (Network set as Wilder), this will be the same as the subdomain in the header and other flows where we display the name. What do you think, should we remove the root segment if REACT_APP_NETWORK is set? 
<img width="793" alt="Screenshot 2022-06-06 at 23 08 20" src="https://user-images.githubusercontent.com/39112648/172257357-126025a3-a6b0-4269-a31d-47e268a6d8a9.png">

Reply:
Ah ok yeah, this looks right. We probably shouldn't remove the root segment - it might cause confusion if people think they own `0://something` but it's actually `0://wilder.something`